template.yaml

```diff
@@ -0,0 +1,105 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  aws-lambda
+
+  SAM Template for aws-lambda
+
+Globals:
+  Function:
+    Timeout: 20
+
+Resources:
+  S3ListBucketsFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: S3ListBucketsFunction
+      Handler: com.epam.aws.lambda.App::handleRequest
+      Runtime: java11
```

Comment:
It would be better to move Runtime, MemorySize to parameters

Reply:
Fixed